common/app/views/support/EmailHelpers.scala

```diff
@@ -53,12 +53,16 @@ object EmailHelpers {
     s"""<img src="${Static(s"images/email/icons/$name.png")}" class="icon icon-$name">"""
   }
 
-  def img(src: String, alt: Option[String] = None) = Html {
-    s"""<img width="580" class="full-width" src="$src" ${alt.map(alt => s"""alt="$alt"""").getOrElse("")}>"""
+  def imgForArticle(src: String, alt: Option[String] = None) = Html {
+    s"""<img width="${EmailImage.knownWidth}" class="full-width" src="$src" ${alt.map(alt => s"""alt="$alt"""").getOrElse("")}>"""
+  }
+
+  def imgForFront(src: String, alt: Option[String] = None) = Html {
+    s"""<img width="${FrontEmailImage.knownWidth}" class="full-width" src="$src" ${alt.map(alt => s"""alt="$alt"""").getOrElse("")}>"""
```

Comment:
I'm not sure what you're suggesting is a higher order function (it doesn't accept a function as a parameter or return a function), but I agree that this markup could be factored out. Have done so